### ISSUE TITLE
ǲ ligature not in output fonts

### ISSUE THREAD

**Issue by arrowtype** [arrowtype/recursive#343]
**Problem description**

`ǲ` is being set in a fallback font while using Rec Mono Duotone, and getting a .notdef in Font Goggles. 

![image](https://user-images.githubusercontent.com/45946693/80513953-d3601d00-894d-11ea-99da-e67ae61a0168.png)

![image](https://user-images.githubusercontent.com/45946693/80514122-15895e80-894e-11ea-8c40-73568b8274f1.png)

This is likely due to some minor incompatibility. Fix when possible.

**Comment by arrowtype** [comment]
From the [GF latin ext unique glyphs](https://github.com/googlefonts/gftools/blob/81c138c4cc218cf0d1a71e36dd03e71c91e6e3c1/Lib/gftools/encodings/latin-ext_unique-glyphs.nam), these ligatures should be included:

```
0x01F1  Ǳ LATIN CAPITAL LETTER DZ
0x01F2  ǲ LATIN CAPITAL LETTER D WITH SMALL LETTER Z
0x01F3  ǳ LATIN SMALL LETTER DZ
0x01C4  Ǆ LATIN CAPITAL LETTER DZ WITH CARON
0x01C5  ǅ LATIN CAPITAL LETTER D WITH SMALL LETTER Z WITH CARON
0x01C6  ǆ LATIN SMALL LETTER DZ WITH CARON
```

**Comment by arrowtype** [comment]
I could make the Dz (etc), then make the Dzcaron (etc) into component glyphs, but this would be slightly tricky in that the caron is condensed in the Mono versions. So, instead I will fix this in a simpler way.

- [ ] First, make sure these mono glyphs are compatible, with Prepolator Next if possible.
- [ ] Then, duplicate to correct names & unicodes with glyph construction, and remove carons
- [ ] build & check that it has worked. Repeat as necessary.

**Comment by arrowtype** [comment]
Actually, the core glyphs _were_ compatible. Nice!

![image](https://user-images.githubusercontent.com/45946693/82126267-ab5c2080-9779-11ea-8edd-c7c8f3ebd4aa.png)

Also, `dzcaron.italic` _is_ in the rvrn rules, but excluded from the mono rules because it's such a narrow glyph.

![image](https://user-images.githubusercontent.com/45946693/82126302-dcd4ec00-9779-11ea-99d9-7a5e2123109b.png)

**Comment by arrowtype** [comment]
Glyph recipes:

```
DZ = DZcaron | 01F1
Dz = Dzcaron | 01F2
dz = dzcaron | 01F3
```

**Comment by arrowtype** [comment]
Fixed in the latest build! 95c30444dc2e3ff3441c34fe852bb9142672d373

![image](https://user-images.githubusercontent.com/45946693/82132900-08c09380-97b3-11ea-990a-26c895d9f046.png)
